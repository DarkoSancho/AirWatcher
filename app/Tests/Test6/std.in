government official
5
3
3
