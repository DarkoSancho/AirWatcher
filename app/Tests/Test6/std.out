==== RESULTS ====

Change in air quality in the area:  
Sensors used: 5 sensors detected within the radius

|Pollutant|Average BEFORE (µg/m³)|Average AFTER (µg/m³)| Improvement (%) 
|----------|---------------------|---------------------|------------------|
| O3       | 52.6                | 45.1                | -14.3 %          |
| NO2      | 34.2                | 28.7                | -16.1 %          |
| SO2      | 6.8                 | 5.0                 | -26.5 %          |
| PM10     | 24.5                | 20.3                | -17.1 %          |

Conclusion:

ATMO Index BEFORE treatment: 4 (Moderate air quality)  
ATMO Index AFTER treatment : 3 (Fair air quality)

The air quality in the area has improved after the installation of the purifier.